node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/scripts/build

```diff
@@ -112,6 +112,8 @@ if [[ ! -d $OBJ_DIR ]]; then
         --with-sysroot=$TOOLS/$TARGET         \
         --disable-nls                         \
         --disable-shared                      \
+        --disable-target-libsanitizer         \
+        --disable-libsanitizer                \
```

Comment:
it's a tool similar to valgrind. it's an optional utility to ensure you're not going out of the memory area and for memory leaks. it's not used